projects/dashboards/gfbio/src/app/app.component.html

```diff
@@ -65,6 +65,10 @@
             </ng-container>
         </geoengine-map-container>
     </div>
+    <geoengine-map-info
```

Comment:
this gets covered up if there are too many layers

Reply:
![image](https://github.com/geo-engine/geoengine-ui/assets/10283892/92b0ffa1-f46c-44f1-a728-a25813ec59f3)

Reply:
https://github.com/geo-engine/geoengine-ui/pull/720/commits/4206dae5ccb78469753122cd3e8a3e24564861f8